### ISSUE TITLE
Postman Test

### ISSUE THREAD

**Issue by denmark-dangnagui** [LarryDpk/Google-Bard#11]
I'm working on it using your library. By the way, I want to test with Postman, so I tested url, header values, and body values based on logs, but there is a 404 error, is there a way to test with Postman?

**Comment by LarryDpk** [comment]
How did integrate the lib in your service? Did you integreate with Spring Boot as a web application?

**Comment by denmark-dangnagui** [comment]
Yes, I'm working on a spring maven right now. i'm working on it by adding dependencies. So I want to test it on Postman while analyzing the code and taking debugs and logs, so I'm trying it out, but it's not working. :(

**Comment by LarryDpk** [comment]
Can you share your code?

**Comment by denmark-dangnagui** [comment]
yes i can. how can i share my code? honestly, i didn't modify any code of yours yet.

**Comment by denmark-dangnagui** [comment]
Method : POST
URL : https://bard.google.com/_/BardChatUi/data/assistant.lamda.BardFrontendService/StreamGenerate?rt=c&_reqid=101020&bl=boq_assistant-bard-web-server_20230514.20_p0
Í›
Headers : 
<img width="505" alt="스크린샷 2023-06-18 오후 9 09 38" src="https://github.com/LarryDpk/Google-Bard/assets/104557007/dc585618-970f-415d-be4f-98665f7954dd">
BODY
<img width="494" alt="스크린샷 2023-06-18 오후 9 10 17" src="https://github.com/LarryDpk/Google-Bard/assets/104557007/aae771a5-ae57-4285-9a2f-b53416f19cd2">

can you tell me what the problem is?

**Comment by LarryDpk** [comment]
So you just called to google bard, didn't use the lib, right?

**Comment by denmark-dangnagui** [comment]
Oh, I said it wrong, I got the code you uploaded and ran it as it was.

Well, I used the library. I think I used it when I sent a request from Intellij.

So I'm analyzing how the code you made sends the request.
That's why I took a log and ran it as a postman.
I took the log and it came out like that.
I've tried sending a request as a postman to the API I made.

But can't I test it on Postman without sending a request to the API I made?

The sample code you uploaded says that if you run the method with ask(), the response value will come out, so can't i test it in postman without using ask()?
